### PR TITLE
added home_page_app

### PR DESCRIPTION
**Pull request by Brianktoo**
add `home_page_app feature` to display our landing page

### CODE REVIEW

**Comment by Brianktoo** [comment]
kindly review this for me

File: flask-contact-form-app/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*/__pycache__/
```
Comment (codetaskmaster) [inline flask-contact-form-app/.gitignore:1]: Add whitespace!

**Review by codetaskmaster — CHANGES_REQUESTED** [review]

File: flask-contact-form-app/.gitignore (hunk 1 above)
Comment (Brianktoo) [reply flask-contact-form-app/.gitignore:1]: done

**Comment by Brianktoo** [comment]
add `home_page_app feature`

**Pull request merged**